client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+    <div>
+        <b-row align-v="center">
+            <b-col v-if="display == 'radio' || display == 'radiobutton'">
+                <b-form-group>
+                    <!-- Select single from radio -->
+                    <b-form-radio-group v-model="currentValue" :options="formattedOptions" text-field="label" stacked>
+                    </b-form-radio-group>
+                </b-form-group>
+            </b-col>
+            <b-col v-else-if="display == 'checkboxes'">
+                <!-- Select multiple from checkboxes -->
+                <b-form-group>
+                    <b-form-checkbox-group
+                        v-model="currentValue"
+                        :options="formattedOptions"
+                        value-field="value"
+                        text-field="label"
+                        stacked>
+                    </b-form-checkbox-group>
+                </b-form-group>
+            </b-col>
+            <b-col v-else>
+                <!-- Multiple select from drop down -->
+                <multiselect
+                    v-if="multiple"
+                    v-model="currentValue"
+                    :options="formattedOptions"
+                    :multiple="true"
+                    placeholder="Select value"
+                    deselect-label=""
+                    select-label=""
+                    track-by="value"
+                    label="label">
+                </multiselect>
+                <!-- Single select from drop down -->
+                <multiselect
+                    v-else
+                    v-model="currentValue"
+                    :options="formattedOptions"
+                    deselect-label=""
+                    select-label=""
+                    label="label"
+                    track-by="value">
+                    {{ currentValue }}
+                </multiselect>
+            </b-col>
+        </b-row>
+    </div>
+</template>
+
+<script>
+import Multiselect from "vue-multiselect";
+
+export default {
+    components: {
+        Multiselect,
+    },
+    props: {
+        value: {
+            default: null,
+        },
+        defaultValue: {
+            type: [String, Array],
+            required: true,
+        },
+        options: {
+            type: Array,
+            required: true,
+            default: undefined,
```

Comment:
As mentioned before, there is no need to declare `default` and `required` at the same time.

Reply:
@guerler Agreed - this change request was just accidentally no copied over. You should see it there now.